src/utils/optionsMessage.js

```diff
@@ -1,24 +1,22 @@
 /**
-  * @param {Character} player Contains the player's information
+  * @param {Player} player Contains the player's information
+  * @param {Character} character Contains the character's information
   * @returns {string} A message to display detailing possible user actions
   */
-export const optionsMessage = ({ player }) => {
-  const skillDescriptions = () => {
-    let listOptions = '';
-    for (let i = 0; i < player.skills.length; i++) {
-      let option = player.skills[i];
-      listOptions += `
+export const optionsMessage = (player, character) => {
```

Comment:
Seems like `player` does not get used whoops. I think I was going to change `PLAYER!` to be the player's name, but there is no name on `player`. We can update this later to do that...